distribution/lib/Standard/Table/0.0.0-dev/src/IO/Auto_Detect.enso

```diff
@@ -0,0 +1,36 @@
+from Standard.Base import Any, Problem_Behavior, Nothing, Error, Panic, Meta, File, File_Format, Plain_Text, Bytes
+from Standard.Base.Error.Common import Unsupported_File_Type_Data, No_Such_Method_Error_Data, Illegal_Argument_Error_Data
+
+from Standard.Table.Delimited.Delimited_Format import Delimited
+from Standard.Table.Excel.Excel_Format import Excel
+
+types = [Plain_Text, Delimited ',', Excel, Bytes]
```

Comment:
```suggestion
## PRIVATE
known_file_types = [Plain_Text, Delimited ',', Excel, Bytes]
```

I'd make the name a bit more meaningful, and I think we should either add a comment explaining what this is OR a comment `PRIVATE` so it is kept private.